src/main/java/fr/insee/genesis/infrastructure/model/document/schedule/ScheduleDocument.java

```diff
@@ -0,0 +1,34 @@
+package fr.insee.genesis.infrastructure.model.document.schedule;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(collection = "schedules")
+public class ScheduleDocument {
```

Comment:
I don't like the name ScheduleDocument here, it's mostly a survey description... Or something like this...